Plots/XY/NCL_xy_16.py

```diff
@@ -0,0 +1,108 @@
+"""
+NCL_xy_16.py
+===============
+This script illustrates the following concepts:
+   - Drawing a legend inside an XY plot
+   - Drawing an X reference line in an XY plot
+   - Reversing the Y axis
+   - Using log scaling and explicit labeling
+   - Changing the labels in a legend
+   - Creating a vertical profile plot
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/xy_16.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/xy_16_1_lg.png
+                         https://www.ncl.ucar.edu/Applications/Images/xy_16_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+from matplotlib.ticker import (ScalarFormatter, NullFormatter)
+import matplotlib.ticker as tic
+
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/atmos.nc"), decode_times=False)
+U = ds.U.isel(time=0).drop('time').isel(lon=0).drop('lon')
+
+# Extract slices of the data at different latitudes using the index of the desired value
+U20 = U.isel(lat=39).drop('lat')
+U30 = U.isel(lat=42).drop('lat')
+U40 = U.isel(lat=46).drop('lat')
+U50 = U.isel(lat=49).drop('lat')
+
+###############################################################################
+# Plot with linear y axis:
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(figsize=(8, 8))
+ax = plt.axes()
+
+# Format axes limits, ticks, and labels
+gvutil.set_axes_limits_and_ticks(ax, xlim=(-20, 40), ylim=(1000, 0), xticks=np.arange(-20,60,10), yticks=np.arange(0,1200,200))
```

Comment:
Minor formatting error- add spaces to lists
```suggestion
gvutil.set_axes_limits_and_ticks(ax, xlim=(-20, 40), ylim=(1000, 0), xticks=np.arange(-20, 60, 10), yticks=np.arange(0, 1200, 200))
```